tests/bundle/Component/TemplateComponentTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Bundle\TwigComponents\Component;
+
+use Ibexa\TwigComponents\Component\TemplateComponent;
+use PHPUnit\Framework\TestCase;
+use Twig\Environment;
+
+final class TemplateComponentTest extends TestCase
+{
+    public function testRenderWithParameters(): void
```

Comment:
I would also add a scenario where a new parameter is appended to the existing ones.

Reply:
Added additional parameter test in [b0f1fc2](https://github.com/ibexa/twig-components/pull/13/commits/b0f1fc25b3c167d316c4976773313a0003ee1683)